Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -414,73 +403,118 @@ function Wait-ServiceFabricApplicationTypeTerminalStatus
         -RetryMessage (Get-VstsLocString -Key SFSDK_RetryingGetApplicationType)
 }
 
-function Unregister-ServiceFabricApplicationTypeAction
+function Wait-ServiceFabricApplicationTypeRegistrationStatus
 {
-    Param(
+    Param (
         [string]
         $ApplicationTypeName,
 
         [string]
         $ApplicationTypeVersion,
 
         [int]
-        $TimeoutSec
+        $TimeoutSec,
+
+        [string]
+        $TimeoutAction,
+
+        [string]
+        $OperationType
```

Comment:
Same as last time - i dont see much value if you need to pass the caller information to make decisions in the common method

Reply:
Okay